content/en/database_monitoring/setup_mysql/rds.md

```diff
@@ -0,0 +1,346 @@
+---
+title: Setting Up Database Monitoring for Amazon RDS managed MySQL
+kind: documentation
+description: Install and configure Database Monitoring for MySQL managed on Amazon RDS.
+further_reading:
+- link: "/integrations/mysql/"
+  tag: "Documentation"
+  text: "Basic MySQL Integration"
+
+---
+
+{{< site-region region="us3,gov" >}}
+<div class="alert alert-warning">Database Monitoring is not supported for this site.</div>
+{{< /site-region >}}
+
+Database Monitoring provides deep visibility into your MySQL databases by exposing query metrics, query samples, explain plans, connection data, system metrics, and telemetry for the InnoDB storage engine.
+
+The Agent collects telemetry directly from the database by logging in as a read-only user. Do the following setup to enable Database Monitoring with your MySQL database:
+
+1. [Configure database parameters](#configure-mysql-settings)
+1. [Grant the Agent access to the database](#grant-the-agent-access)
+1. [Install the Agent](#install-the-agent)
+1. [Install the RDS integration](#install-the-rds-integration)
+
+## Before you begin
+
+Supported MySQL versions
+: 5.6, 5.7, or 8.0+
+
+Supported Agent versions
+: 7.30.0+
+
+Performance impact
+: The default Agent configuration for Database Monitoring is conservative, but you can adjust settings such as the collection interval and query sampling rate to better suit your needs. For most workloads, the Agent represents less than one percent of query execution time on the database and less than one percent of CPU. <br/><br/>
+Database Monitoring runs as an integration on top of the base Agent ([see benchmarks][1]).
+
+Proxies, load balancers, and connection poolers
+: The Agent must connect directly to the host being monitored, preferably through the instance endpoint. The Agent should not connect to the database through a proxy, load balancer, or connection pooler. While this can be an anti-pattern for client applications, each Agent must have knowledge of the underlying hostname and should stick to a single host for its lifetime, even in cases of failover. If the Datadog Agent connects to different hosts while it is running, the values of metrics will be incorrect.
+
+Data security considerations
+: See [Sensitive information][2] for information about what data the Agent collects from your databases and how to ensure it is secure.
+
+## Configure MySQL settings
+
+Configure the following in the [DB Parameter Group][3] and then **restart the server** for the settings to take effect:
+
+{{< tabs >}}
+{{% tab "MySQL 5.6" %}}
+| Parameter | Value | Description |
+| --- | --- | --- |
+| `performance_schema` | `1` | Required. Enables the [Performance Schema][4]. |
```

Comment:
[4] like isn't rendering for both tables

Reply:
fixed! thank you